packages/nextjs/app/builders/0x8EC90502356657Cc30f22444A8e073FE89322333/page.tsx

```diff
@@ -0,0 +1,62 @@
+import Image from "next/image";
+
+const HomePage = () => {
```

Comment:
Let's use a better const name (instead of HomePage, something like IvanBuilderPage)